.travis.yml

```diff
@@ -1,26 +1,48 @@
 language: scala
 sudo: false
+
+jdk:
+- oraclejdk8
+
 scala:
 - 2.12.6
 - 2.11.12
+
 install:
 - rvm use 2.4.4 --install --fuzzy
 - gem update --system
 - gem install sass
 - gem install jekyll -v 3.8.2
-jdk:
-- oraclejdk8
+
+stages:
+- name: test
+- name: release
+#  if: (branch = master AND type = push) OR (tag IS present)
```

Comment:
We are going to need something like this :)

Reply:
I was just playing around :)